### PR TITLE
fixed problem while uglifying, fixed problem with Weaver.Error, added…

### PR DESCRIPTION
**Pull request by carvilsi**
fixed problem while uglifying
fixed problem with Weaver.Error
added deleting files from object store server

### CODE REVIEW

File: views/weaver/index.html [hunk 1]
```diff
@@ -6,19 +6,19 @@
         <link rel="icon" type="image/png" href="img/favicon.png">
         <script type="application/javascript" src="sdk/weaver-sdk.full.js"></script>
         <script>
-            Weaver.initialize(window.location.href)
+            Weaver.connect(window.location.href)
```
Comment (gvdent) [inline views/weaver/index.html]: Should these changes be part of this PR?

Comment (carvilsi) [inline views/weaver/index.html]: The answer will be: It should not be part of this PR. But we where fixing some stuff of browser and we did it…

Comment (gvdent) [inline views/weaver/index.html]: Fixing stuff is always good... the fact that the error was in BETA is more problematic ;)
